cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -1537,9 +1513,9 @@ features:
     advancedOptions: Advanced options
     Alerts:
       contentHeading: Set alerts for your batch pipeline
-      title: Alerts
+      title: Pipeline alert
```

Comment:
why did this change?

Reply:
That's what currently being used in the UI, but it was hardcoded, so I had to change the corresponding i18n. Not sure when this changed